lib/basedriver/helpers.js

```diff
@@ -323,7 +322,12 @@ async function unzipApp (zipPath, dstRoot, supportedAppExtensions) {
   try {
     logger.debug(`Unzipping '${zipPath}'`);
     const timer = new timing.Timer().start();
-    const extractionOpts = {};
+    /**
+     * @type {ExtractAllOptions}
```

Comment:
is this type notation needed?

Reply:
I'd rather leave a comment here about why we default to system unzip

Reply:
yes, it's needed if you care about intellisense, but it's also wrong.